src/components/Select/SelectBase/index.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { ISelectItemProps, Select as NativeBaseSelect } from 'native-base';
+import { ISelectComponentType } from 'native-base/lib/typescript/components/primitives/Select';
+
+import { theme } from '@theme';
+import { ErrorText, ErrorWrapper } from './styles';
+import { WarningCircle } from 'phosphor-react-native';
+
+type ISelectOptions = {
+  label: string;
+  value: string;
+};
+
+export interface ISelectBaseProps extends Omit<ISelectComponentType, 'Item'> {
+  width?: string;
+  height?: string;
+  options: ISelectOptions[];
+  messageError?: string;
+  error: boolean;
+  selectedValue: string;
+  onChangeValue: (value: string) => void;
+}
+
+export function SelectBase({
+  width = '100%',
+  height = '50',
+  options,
+  messageError,
+  error,
+  onChangeValue,
+  selectedValue,
+  ...rest
+}: ISelectBaseProps) {
+  return (
+    <>
+      <NativeBaseSelect
+        bg={theme.colors.gray_200}
+        borderRadius="35"
```

Comment:
Tirar propriedade do componente ou colocar o "ou" logico recebendo uma propriedade borderRadius para poder editar futuramente em outras telas

Reply:
82939d0128d350675f200f69bf4a67119869fc0a

Reply:
Não foi implementada, só foi colocado como parâmetro da função
 ```suggestion
        borderRadius={"35" || borderRadius}
```

Reply:
correct in [dc526e0](https://github.com/geta-org/sex-adapt-mobile/pull/111/commits/dc526e043efd61f7a632f6cc5d2cacb8216f12fc)